research/flamby/fed_isic2019/fedavg/server.py

```diff
@@ -36,6 +37,28 @@ def main(config: Dict[str, Any], server_address: str, checkpoint_stub: str, run_
     client_manager = SimpleClientManager()
     model = Baseline()
 
+    log(INFO, f"if pretrain: {pretrain}")
+    if pretrain:
```

Comment:
We want to avoid these hard-coded values in our code because someone running this outside of our cluster will have issues. My suggestion would be rather than having a `pretrain: bool` as an argument. You rather have a `pretrain_path: Optional[Path] = None`. If there is a provided path, attempt to load a checkpoint from the provided path.

Reply:
oh these things needed to be deleted, I am shocked they are here